src/test/regress/expected/worker_hash_partition.out

```diff
@@ -25,10 +25,10 @@ SELECT worker_hash_partition_table(:JobId, :TaskId, :Select_Query_Text,
  
 (1 row)
 
-COPY :Table_Part_00 FROM 'base/pgsql_job_cache/job_201010/task_101103/p_00000';
-COPY :Table_Part_01 FROM 'base/pgsql_job_cache/job_201010/task_101103/p_00001';
-COPY :Table_Part_02 FROM 'base/pgsql_job_cache/job_201010/task_101103/p_00002';
-COPY :Table_Part_03 FROM 'base/pgsql_job_cache/job_201010/task_101103/p_00003';
+COPY :Table_Part_00 FROM 'base/pgsql_job_cache/job_201010/task_101103/p_00000.postgres';
```

Comment:
Ok, you need to find Oid of postgres, which seems to be always `10`?

Reply:
Made it automatic using `SELECT usesysid AS userid FROM pg_user WHERE usename = 'postgres' \gset`